proposals/lip-0047.md

```diff
@@ -498,53 +498,194 @@ This module does not define any specific logic for other modules.
+  * Check that the `generatorKey` property of every entry in the `assetPoA.validators` array has length 32.
+  * Check that the `blsKey` property of every entry in the `assetPoA.validators` array has length 48.
+  * Check that the `proofOfPossession` property of every entry in the `assetPoA.validators` array has length 96.
 
-#### After Genesis Block Execution
+* Checks for the `assetPoA.snapshotSubstore` (note that these checks are equivalent to those for the verification of the Authority Update command): 
+  * Check that the array has at least 1 element and at most `MAX_NUM_VALIDATORS` elements.
+  * Check that the `address` properties of entries in the `assetPoA.snapshotSubstore.activeValidators` are pairwise distinct.
+  * Check that entries in the `assetPoA.snapshotSubstore.activeValidators` array are ordered lexicographically according to `address`.
+  * Check that for every element `activeValidator` in the `assetPoA.snapshotSubstore.activeValidators` array, there is an entry with `address == activeValidator.address` in the `assetPoA.validators` array.
+  * Check that the `weight` property of every entry in the `assetPoA.snapshotSubstore.activeValidators` array is a positive integer.
+  * Let `totalWeight` be the sum of the `weight` property of every entry in `assetPoA.snapshotSubstore.activeValidators`. Then, check that `totalWeight` is less than or equal to `MAX_UINT64`.
+  * Check that the value of `assetPoA.snapshotSubstore.threshold` is within the following range:
+     * Minimum value: ⌊ ⅓ × `totalWeight`⌋ + 1, where ⌊⋅⌋ is the floor function.
+     * Maximum value: `totalWeight` 
 
-After the genesis block `g` is executed, the following logic is executed:
+* Create an entry in the validator substore for each entry `validator` in the `assetPoA.validators` array as:
+  * `storeKey`: `validator.address`.
+  * The `storeValue` is the serialization of the object `validatorObject` following `validatorObjectSchema` with `validatorObject.name = validator.name`. 	
+
+* Create an entry in the name substore for each entry `validator` in the `assetPoA.validators` array as:
+  * `storeKey`: `validator.name` serialized as a utf-8 encoded string.
+  * The `storeValue` is the serialization of the object `validatorAddress` following `validatorAddressSchema` with `validatorAddress.address = validator.address`.
+
+* Create three entries in the snapshot substore as:
+  * The `storeKey` of each of the entries are `uint32be(0)`, `uint32be(1)`, and `uint32be(2)` respectively.
+  * The `storeValue` is the same for the three entries and equal to the serialization of `assetPoA.snapshotSubstore`.
+
+* Create an entry in the chain properties substore as:
+  * The `storeValue` is the serialization of a `chainProperties` object following `chainPropSchema` where:
+    * `chainProperties.roundEndHeight` = `g.header.height`.
+    * `chainProperties.validatorsUpdateNonce` = 0.
+
+##### Genesis PoA Store Schema
+
+```java
```

Comment:
I would suggest to run `prettier` on the schema to improve the spacing.

Reply:
good point, done